nodes/src/nodes/tool_filesystem/README.md

```
@@ -0,0 +1,48 @@
+| Write files        | Enable `write_file`                                                                                               |
+| List directories   | Enable `list_directory`                                                                                           |
+| Create directories | Enable `create_directory`                                                                                         |
+| Stat (metadata)    | Enable `stat_file`                                                                                                |
+| Delete files       |                                                                                                                   |
+| Path Whitelist     | Optional regex patterns. If non-empty, every operation's path must match at least one pattern. Empty = allow all. |
+
+## Storage location
+
+Files land under the configured storage backend (defaults to `~/.rocketlib/store/`). For the default filesystem backend the absolute path is:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)